src/main/java/org/example/models/UserInterface.java

```diff
@@ -0,0 +1,181 @@
+package org.example.models;
+
+import org.example.models.Doctor;
+import org.example.models.MedicalVisit;
+import org.example.models.Patient;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.SocketHandler;
+
+public class UserInterface {
+    public void mainMenu() {
+        System.out.println("(1)Doctor or (2)Patient ?");
+    }
+
+    public void chooseUser() {
+        System.out.println("Enter your id");
+    }
+
+    public void actionsMenudoctor() {
+        System.out.println("1. Add visit date.");
+        System.out.println("2. Delete visit.");
+        System.out.println("3. Cancel visit.");
+        System.out.println("4. Review all visits");
+    }
+
+    public void actionsMenupatient() {
+        System.out.println("1. Review possible dates of visit");
+        System.out.println("2. Reserve a visit");
+        System.out.println("3. Search for possible visit dates");
+    }
+
+    public int getValue() {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Choice?");
+        int number = scan.nextInt();
+        return number;
+    }
+
+    public void Paclist(List<Patient> patients) {
+        for (int i = 0; i < patients.size(); i++) {
+            System.out.println(patients.get(i).toString());
+        }
+    }
+
+    public void Doclist(List<Doctor> Mds) {
+        for (int i = 0; i < Mds.size(); i++) {
+            System.out.println(Mds.get(i).toString());
+        }
+    }
+
+    public void Visitlist(List<Doctor> Mds, List<Patient> patients, List<MedicalVisit> visits, int docID, boolean isDoctor) {
+        for (int i = 0; i < visits.size(); i++) {
+            if (docID == visits.get(i).getDoctorId()) {
+                System.out.println(visits.get(i).toString());
+            }
+        }
+    }
+
+
+    public void run(List<Patient> patients, List<Doctor> Mds, List<MedicalVisit> visits) {
```

Comment:
czy ta funkcja ma być w main w pętli? Jak to ma działać?

Reply:
ścieżka lekarza i pacjenta działa w petli